examples/ecr.tf

```diff
@@ -28,3 +30,40 @@ resource "kubernetes_secret" "example_team_ecr_credentials" {
   }
 }
 
+########SLACK NOTIFIATIONS OF ECR SCAN RESULTS########################
+
+#To send notifications to slack of the ECR scanned results, you may insert the following two modules, one creates the lambda function.
+# and the other creates the even bridge that forwards the result details to the lambda function
+
+# SLACK KUBERNETES SECRET 
+
+# Note that you will need to create a kubernetes secret that contains the slack token and the ECR repo.
+
+# The lambda needs the slack token and repo corresponding to the channel. The keys are set as environment variables. This secret needs
+# be created with the following keys:
+
+#Key 1: repo (without the pre-fix e.g if the url is 754256621582.dkr.ecr.eu-west-2.amazonaws.com/webops/webops-ecr1:rails, then in this case you need to supply 'webops/webops-ecr1')
+#Key 2: token
+
+# You can run the following kubernets example command to the create the secret: 
+
+# kubectl create secret generic <SECRET_NAME> --from-literal=token=<SLACK_TOKEN> --from-literal=repo=<ECR_REPO> -n <NAMESPACE>
+
+data "kubernetes_secret" "slack_cred" {
```

Comment:
I think it would be  clearer to have a line something like:

> Once you have created the kubernetes secret containing your slack token, you need to provide terraform with a reference to it like this:

Reply:
this data resource block is now moved to the lambda function itself.